pilota/Cargo.toml

```diff
@@ -38,6 +38,9 @@ criterion = "0.5"
 proptest = "1"
 rand = "0.8"
 
+[features]
+unstable = []
```

Comment:
What is this for?

Reply:
Clippy warned of features that are used and not exist in Cargo.toml.

We have used this feature in assert_remaining macro:
![image](https://github.com/cloudwego/pilota/assets/3314335/9f71e0f7-5c1c-4b23-a795-4f7ab7fe73db)